src/pages/workspace/companyCards/WorkspaceCompanyCardPageEmptyState.tsx

```diff
@@ -71,7 +71,7 @@ function WorkspaceCompanyCardPageEmptyState({policy}: WithPolicyAndFullscreenLoa
                 illustrationStyle={styles.emptyStateCardIllustration}
                 illustrationContainerStyle={[styles.emptyStateCardIllustrationContainer, styles.justifyContentStart]}
                 titleStyles={styles.textHeadlineH1}
-                isButtonDisabled={workspaceAccountID > CONST.DEFAULT_NUMBER_ID}
+                isButtonDisabled={workspaceAccountID === CONST.DEFAULT_NUMBER_ID}
```

Comment:
Sorry for the hassle, I missed this when updating the approach last minute. Tested and works well 🎉 The button is disabled when the feature was enabled offline, that means the workspaceAccountID is not present yet (for older policies - new collect policy would have the accountID after creation)

https://github.com/user-attachments/assets/516f4448-56e4-4afd-a748-a498c8480b69